src/api/java/blusunrize/immersiveengineering/api/crafting/ClocheRenderFunction.java

```diff
@@ -31,6 +32,12 @@
 
 	Collection<Pair<BlockState, TransformationMatrix>> getBlocks(ItemStack stack, float growth);
 
+	// hook for dynamic cloche recipes
+	default void injectQuads(ItemStack stack, float growth, Consumer<?> quadConsumer)
+	{
+		// default behaviour: do nothing
```

Comment:
You could drop this, it's pretty obvious that it does nothing =P

Reply:
You mean the comment I suppose?

Reply:
Indeed. Method stubs look better when empty, especially given that the comment *above* the method describes what it could do.